vars/all_var.yml

```diff
@@ -1,70 +1,11 @@
 ---
-# File: all_var.yml
-# Created: 4/4/17
-#
-# Incorporates encrypted vars/common.yml and vars/{{ env }}/env.yml
-# Include all environment variables here.
-# reference source variables from appropriate file:
-# - env_ = environment specific vairables from .env{{ env }}/env.yml
-# - common_ = stored in ./common.yml
-# - vault_env_ = stored in encrypted vault in ../vault/env/{{ env }}/env.yml
-# - vault_ = stored in encrypted vault in ../vault/common.yml
-# VARS Files can't be loaded inside vars files.
-# So the following files need to be included in each playbook
-#  var_files:
-#  - import_tasks: "./common.yml"
-#  - import_tasks: "./env/{{ env }}/env.yml"
-#  - import_tasks: "../vault/env/{{ env }}/vault.yml"
-
-  ami_release: "{{ env_ami_release }}"
-
-      # This image should be updated monthly.
-      # Current version is: EAST-RH 7-3 Gold Image V.1.09 (HVM) 04-21-2017 - ami-fdde41eb
-      # EAST-RH 6-9 Gold Image V.1.01 (PV) 04-21-17 - ami-3dda452b
-      # EAST-RH 6-8 Gold Image V.3.17 (PV)Splunk-CFN-FIPS140-02-24-17 (ami-f070abe6)
-      # EAST-RH 7-3 Gold Image V.1.10 (HVM) 05-18-17 - ami-3aff872c (previous)
-      # EAST-RH 7-4 Gold Image V.1.15 (HVM) 11-2-17 - ami-5a6ac120
-
-  ami_app_gold_image: "{{ env_ami_app_gold_image }}" # Change the AMI, from which you want to launch the server
 
   ### Environment:
   env: "{{ env_env }}"
 
-  ### Cloud Formation Tags:
-  cf_tags_business: "{{ common_cf_tags_business }}"
-  cf_tags_application: "{{ common_cf_tags_application }}"
-
-
-  # Make Python and mod_wsgi version parameters
-  mod_wsgi_ver_full: "{{ env_mod_wsgi_ver_full }}"
-  mod_wsgi_processes: "{{ env_mod_wsgi_processes | default(1) }}"
-  mod_wsgi_threads: "{{ env_mod_wsgi_threads | default(15) }}"
-  mod_wsgi_timeout: "{{ env_mod_wsgi_timeout | default(300) }}"
-
-
-  # pip install extra arguments needed:
-  # when FIPS-140.2 is enabled MD5 checksums are not supported.
-  # This causes pip to fail.
-  # check file: /proc/sys/crypto/fips_enabled
-  # 1 = Enabled | 0 = disabled
-  # If enabled we need to pass -i https://pypi.org/simple/ to extra_args
-  # in ansible pip command
-  pip_extra_args: " -i https://pypi.org/simple/ "
-
-
-  # Remote user access account
-  # [JZ] - This is not needed, each env has the same emore user account, moving to common.yml
-  #remote_user_account: "{{ env_remote_user_account }}"
-  #remote_admin_account: "{{ env_remote_admin_account }}"
-
   ### DJANGO_SECRET_KEY:
-  # 8 Character Prefix and Suffix
-  # Relate Suffix to Deployment Environment
-  secret_pfx: "{{ env_secret_pfx }}"
-  secret_sfx: "{{ env_secret_sfx }}"
   # 50 character Random Key
   # http://www.miniwebtool.com/django-secret-key-generator/
-  rand_secret: "{{ env_rand_secret }}"
   django_secret_key: "{{ env_django_secret_key }}"
```

Comment:
Important to note that secret_pfx, secret_sfx, and rand_secret are used only in the individual env.yml files to construct the final django_secret_key, so only django_secret_key is needed in the all vars file.